web/src/pages/apd/proposed-budget/BudgetSummary.js

```diff
@@ -41,22 +34,37 @@ DataRow.propTypes = {
   title: PropTypes.string.isRequired
 };
 
-const DataRowGroup = ({ data, year }) => (
-  <Fragment>
-    {categories.map(({ category, title }, i) => (
-      <DataRow
-        category={category}
-        data={data[category][year]}
-        key={i} // eslint-disable-line react/no-array-index-key
-        title={title}
-      />
-    ))}
-  </Fragment>
-);
+const DataRowGroup = ({ data, year, apdType }) => {
+  const categories = [
+    { category: 'statePersonnel', title: 'State Staff' },
+    { category: 'expenses', title: 'Other State Expenses' },
+    { category: 'contractors', title: 'Private Contractor' },
+    { category: 'combined', title: 'Subtotal' }
+  ];
+  apdType === 'MMIS'
+    ? categories.splice(0, 0, {
```

Comment:
This totally works, so no need to change! Just an FYI, the function `unshift` adds to the beginning of the array by default so you don't need to specify the position

Reply:
ohhhhhh, I like that